eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/Serialisation/Generators.hs

```diff
@@ -101,7 +114,11 @@ instance
   ) =>
   Arbitrary (AlonzoAuxiliaryData era)
   where
-  arbitrary = AlonzoAuxiliaryData <$> arbitrary <*> arbitrary
+  arbitrary =
+    frequency
+      [ (9, AlonzoAuxiliaryData <$> arbitrary <*> arbitrary),
+        (1, pure $ AlonzoAuxiliaryData mempty mempty)
```

Comment:
I think I'll actually just remove those. I added these cases because I noticed that the `Arbitrary` instance for `Seq` doesn't seem to generate empty sequences. It might just be that it generates them very rarely, but either way the issue seems to be with `Seq` not `AlonzoAuxiliaryData`.